sdk/cosmos/azure-cosmos/azure/cosmos/aio/_cosmos_client_connection_async.py

```diff
@@ -2812,6 +2813,11 @@ def __GetBodiesFromQueryResult(result: Dict[str, Any]) -> List[Dict[str, Any]]:
                 documents._OperationType.QueryPlan if is_query_plan else documents._OperationType.ReadFeed
             )
             headers = base.GetHeaders(self, initial_headers, "get", path, id_, typ, options, partition_key_range_id)
+
+            change_feed_state = options.get("changeFeedState", None)
+            if change_feed_state and isinstance(change_feed_state, ChangeFeedState):
```

Comment:
change_feed_state should always be an instance of ChangeFeedState, just add a check here to prevent unintentional use of "changeFeedState" for other type/purpose